client/task_runner.go

```diff
@@ -870,6 +883,8 @@ func (r *TaskRunner) run() {
 			select {
 			case success := <-prestartResultCh:
 				if !success {
+					//FIXME is this necessary?
```

Comment:
It is since you call restart through restarts so if it fails you may have previously created stuff